stories/molecules/RadioButton.tsx

```diff
@@ -10,11 +10,12 @@ export default {
 export const _RadioButton = (args) => (
     <RadioButtonGroup name="my-group">
         <RadioButton {...args} />
-        <RadioButton value="option2" id="radio-2">
+        <RadioButton id="radio-2" {...{ value: 'option2' }}>
```

Comment:
Why did you rewrite option prop?

Reply:
It shows error:
 Type '{ children: string; id: string; value: string; }' is not assignable to type 'IntrinsicAttributes & Props & RefAttributes<HTMLElement>'.   Property 'value' does not exist on type 'IntrinsicAttributes & Props & RefAttributes<HTMLElement>'.

As i understand value will pass like rest object <div {...block({ ...rest })}>